config/packages/surfnet_saml.yaml

```diff
@@ -4,6 +4,8 @@ parameters:
 surfnet_saml:
   enable_authentication: true
   hosted:
+    attribute_dictionary:
+      ignore_unknown_attributes: true
```

Comment:
If this fixes the problem then that's a step in the right direction.

I do think the old behavior was that unknown attributes were *not* ignored as that is the default behavior of the saml bundle.

1. Might be better to fix this issue at the source. In other words, allow the property to not be configured, and default it to `false`?

Reply:
Yes, that is fine by me. Would you like me to make a PR for that?

Reply:
Yes please